src/sources/audio_base_latency.test.ts

```diff
@@ -0,0 +1,38 @@
+import { getBrowserVersion, isGecko, isWebKit } from '../../tests/utils'
+import getAudioBaseLatency from './audio_base_latency'
+
+describe('Sources', () => {
+  describe('audio base latency', () => {
+    it("doesn't fail", () => {
+      const result = getAudioBaseLatency()
+      if (!hasBaseLatencySupport()) {
+        expect(result).toBe(undefined)
+        return
+      }
+
+      if (isGecko()) {
+        expect(result).toBe(0)
+      } else {
+        expect(result).toBeGreaterThan(0)
+      }
+    })
+
+    it('returns a stable value', () => {
+      const first = getAudioBaseLatency()
+      const second = getAudioBaseLatency()
+      expect(second).toBe(first)
+    })
+  })
+})
+
+function hasBaseLatencySupport() {
+  if (isWebKit()) {
+    const { major, minor } = getBrowserVersion() || { major: 0, minor: 0 }
+    if (major < 14) {
+      return false
+    } else {
+      return minor >= 1
+    }
```

Comment:
Looks like the condition is incorrect. Versions 15.0, 16.0 and so on will give `false`.